### PR TITLE
Front 88

### PR DESCRIPTION
**Pull request by Manizha-khorram**
I created base URL for when the user is in localhost, it access the local server. If not it will access the Render.com URL.

### CODE REVIEW

File: src/util/fetch.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { error } from 'console'
 // Exported cookie name to easily use the same name for the same cookie accross all application to avoid misprints
 import { getCookie , setCookie, deleteCookie, cookieName } from '../util/Authentication';
 
+//Export the server URL
+export const baseURL = window.location.hostname === 'localhost'? 'http://localhost:8000/api/v1/': 'https://shelf-share.onrender.com'
```
Comment (jpremo) [inline src/util/fetch.ts:6]: shouldn't the render.com link have /api/v1 in it as well?

Comment (larasodjati) [reply src/util/fetch.ts:6]: @jpremo as far as I know we need to add it by ourselves, that's what I did when setup documentation and testing the prod URL in Postman.

Comment (Manizha-khorram) [reply src/util/fetch.ts]: @jpremo I got confused first, but thanks for the comment.
@larasodjati, if we dont add it, it shows route dont exist so we need to add it to access like books or messages ...

Comment (larasodjati) [reply src/util/fetch.ts]: @Manizha-khorram true! Thats why we need to add it manually like what I said before, unless there's a setting to add it automatically during the deployment. @jpremo could you please approve this?

**Review by jpremo — APPROVED** [review]

**Pull request merged**